packages/cli/src/controller/deploy-controller.ts

```diff
@@ -185,11 +193,145 @@ export async function imageVersions(name: string, version: string, authToken: st
     errorHandle(e, 'Failed to get image:');
   }
 }
+
 export function splitEndpoints(endpointStr: string): string[] {
   return endpointStr.split(',').map((e) => e.trim());
 }
+
 export interface EndpointType {
   network: string;
   endpoint: string;
   chainId: string;
 }
+
+export function splitMultichainDataFields(fieldStr: string): multichainDataFieldType {
+  let result: multichainDataFieldType = {};
+
+  splitEndpoints(String(fieldStr)).forEach(unparsedRow => {
+    let regexpResult: string[] = unparsedRow.match(/(.*?):(.*)/);
+    if (regexpResult) {
+      regexpResult = Object.values(regexpResult);
+      if (regexpResult && regexpResult.length == 6 && ['http','https','ws','wss'].indexOf(regexpResult[1])===-1 ) {
+        result[regexpResult[1]] = regexpResult[2];
+      }
+    }
+  });
+
+  return result;
+}
+
+export function default_command_flags() {
```

Comment:
```suggestion
export function defaultCommandFlags() {
```

It would be good to tie the return type of this to `deploymentFlagsInterface`, currently they could easily diverge